<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRef</class>
 <widget class="QDialog" name="DialogRef">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="showRef">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseRef">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="showTar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseTar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QListWidget" name="showRefSeq">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="showTarSeq">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelRefDir">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selet Refence Image Folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRefList">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selet Refence Image Sequence</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTarList">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selet Moving Image Sequence</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTarDir">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selet Moving Image Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseOutput">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="showOutput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelOutputDir">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selet Output Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLog">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRefSelected">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTarSelected">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
